reana_server/status.py

```diff
@@ -407,6 +424,13 @@ def get_memory_usage(self):
 
         return result
 
+    def get_available_memory(self):
+        """Get list of available nodes memory."""
+        nodes = self.get_memory_usage()
+        if not nodes or "error" in nodes:
+            return self.get_nodes_memory()
```

Comment:
In case of any errors to get available nodes memory, total memory from all nodes is returned simply not to stop any workflows from passing into the cluster, since we don't know if it's free or not..

Reply:
We can add a FIXME to revisit it once the new Kubernetes version is released.

Reply:
In this case it wouldn't help to wait until Kubernetes release, since `get_memory_usage()` could also fail while calling `metrics.k8s.io` API. So we would still need to handle these kind of errors.

Current logic to check if workflow could be started in Kubernetes (executed in `reana_ready()`):
```max_node_available_memory >= workflow_min_job_memory```
So returning all the nodes memory in this case would mean to start all the incoming workflows